src/jdk.crypto.cryptoki/share/native/libj2pkcs11/p11_keymgmt.c

```diff
@@ -920,6 +926,9 @@ JNIEXPORT jlong JNICALL Java_sun_security_pkcs11_wrapper_PKCS11_C_1DeriveKey
     case CKM_TLS12_MASTER_KEY_DERIVE:
         tls12CopyBackClientVersion(env, ckpMechanism, jMechanism);
         break;
+    case CKM_NSS_TLS_EXTENDED_MASTER_KEY_DERIVE:
+        tlsEmsCopyBackClientVersion(env, ckpMechanism, jMechanism);
+        break;
```

Comment:
Do we need a case for `CKM_NSS_TLS_EXTENDED_MASTER_KEY_DERIVE_DH`?

Reply:
In the NSS source code, there is a comment in `ssl3con.c` saying, when use the DH derivation modes, we need to use `CKM_***_DH` mechanism which does not expect to return the version. So, here when copy back client version, I don't think we need a case for `CKM_NSS_TLS_EXTENDED_MASTER_KEY_DERIVE_DH`.